### PR TITLE
Enable audio video streaming

### PR DESCRIPTION
**Pull request by anosach-luxoft**

### CODE REVIEW

**Comment by anosach-luxoft** [comment]
@AGaliuzov, @malirod, @pestOO, please review

**Comment by AGaliuzov** [comment]
reviewed

File: src/components/media_manager/src/pipe_streamer_adapter.cc [hunk 1]
```diff
@@ -64,54 +58,51 @@ PipeStreamerAdapter::PipeStreamer::~PipeStreamer() {
...
-  //close(pipe_fd_);
-  //unlink(named_pipe_path_.c_str());
+  pipe_.Close();
 }
 
 bool PipeStreamerAdapter::PipeStreamer::Send(
     protocol_handler::RawMessagePtr msg) {
   LOG4CXX_AUTO_TRACE(logger);
-  /*ssize_t ret = write(pipe_fd_, msg->data(), msg->data_size());
+  size_t ret = pipe_.Write(reinterpret_cast<const char*>(msg->data()),
```
Comment (pestOO) [inline src/components/media_manager/src/pipe_streamer_adapter.cc:92]: Could we modify interface of Write with a `uin8_t` type and use any cast deeply?

**Comment by pestOO** [comment]
reviewed

File: src/components/media_manager/src/pipe_streamer_adapter.cc (hunk 1 above)
Comment (anosach-luxoft) [inline src/components/media_manager/src/pipe_streamer_adapter.cc:92]: @pestOO, I prefer to use common way with `const char*` passing. We also use `const char*` in socket wrapper for example

**Pull request merged**